sklearn/ensemble/forest.py

```diff
@@ -374,6 +374,20 @@ def feature_importances_(self):
         return sum(all_importances) / len(self.estimators_)
 
 
+# these next two are utility functions for joblib's Parallel. Ideally
+# they would be defined locally in ForestClassifier or ForestRegressor, but
+# joblib complains that it cannot Pickle them when placed there.
+
+def _run_estimator(f, X, out):
+    out += f(X, check_input=False)
+
+
+def _run_estimator2(f, X, out):
```

Comment:
call this `accumulate_prediction_multioutput`, though I'd be interested in merging the single output case in to avoid lots of duplicated logic.